packages/shared/src/__tests__/__examples__/parallel.ts

```diff
@@ -0,0 +1,71 @@
+import { createMachine } from "xstate";
+
+const machine = createMachine(
+  {
+    tsTypes: {} as import("./parallel.typegen").Typegen0,
+    schema: {
+      context: {} as { value: string },
+      events: {} as
+        | { type: "GO_STRING"; answer: string }
+        | { type: "GO_NUMBER"; count: number },
+      services: {} as {
+        jump: {
+          data: boolean;
+        };
+        off: {
+          data: boolean;
+        };
+      },
+    },
+    id: "test",
+    initial: "idle",
+    states: {
+      idle: {
+        on: {
+          GO_STRING: {
+            target: "doesnt",
+          },
+          GO_NUMBER: {
+            target: "works",
+          },
```

Comment:
everything related to this transition is redundant here and somewhat makes analyzing the intent of this test harder